### DISCUSSION TITLE
Allow ⬆️ ⬇️  keyboard navigation in global search results

### DISCUSSION THREAD

**Discussion by JohnONolan** [filamentphp/filament#10578] [Ideas]
You can already navigate through global search results with <kbd>Tab</kbd> and <kbd>Shift</kbd>+<kbd>Tab</kbd> - but it would be great if it was also possible to use ⬆️  and ⬇️  keys to do the same 🕺 

![Screenshot 2023-12-30 at 14 55 52@2x](https://github.com/filamentphp/filament/assets/120485/999c2ab3-f5bd-4a4c-8d51-af5145450ae6)

**Comment by danharrin** [comment]
#10584
